pkg/controller/bundlec/resource_sync_task.go

```diff
@@ -497,6 +479,20 @@ func (st *resourceSyncTask) evalSpec(res *smith_v1.Resource, actual runtime.Obje
 	}
 	obj.SetOwnerReferences(refs)
 
+	// Object GroupKind-specific munging
+	switch obj.GroupVersionKind().GroupKind() {
+	case schema.GroupKind{Group: sc_v1b1.GroupName, Kind: "ServiceInstance"}:
```

Comment:
nit: should Smith have constants for all supported Kinds?

Reply:
Yes, would be good to do this across the whole codebase.